idaes/power_generation/unit_models/tests/test_feedwater_heater_dynamic.py

```diff
@@ -0,0 +1,98 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+
+import pytest
+import pyomo.environ as pyo
+
+from idaes.core import FlowsheetBlock
+from idaes.generic_models.properties import iapws95
+from idaes.power_generation.unit_models import FWH0DDynamic
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+prop_available = iapws95.iapws95_available()
+
+# See if ipopt is available and set up solver
+if pyo.SolverFactory('ipopt').available():
+    solver = pyo.SolverFactory('ipopt')
+else:
+    solver = None
+
+
+@pytest.mark.integration
```

Comment:
We may want to break the tests up so that everything up to the initialization and solve is a unit test the initialization and solve are integration tests.  That will help catch a lot of things before running the integration tests.  I think if you write a test class you can just break up what you have into a sequential set of tests @andrewlee94 may have a recommendation to a good example to follow.

Reply:
The CSTR is always my go-to example.